doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,72 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change the default values, create ``~/.skpkgrc``:
+
```

Comment:
To change the default (A), create -> this is logically not correct. 

The logic should be sth like, "To change the default values prompted when creating a new package, please follow the quick steps below:"